src/borg/remote.py

```diff
@@ -636,6 +636,12 @@ def borg_cmd(self, args, testing):
                 opts.append('--critical')
             else:
                 raise ValueError('log level missing, fix this code')
+            try:
+                borg_logger = logging.getLogger('borg')
+                if borg_logger.json:
+                    opts.append('--log-json')
```

Comment:
Also the entire way of how we configure our logging extensions should maybe redone  -- see comments above regarding `self.emit`.